source/github-events/on-issue-title-field-keydown.tsx

```diff
@@ -0,0 +1,17 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+
+export default function onIssueTitleKeydown(callback: delegate.EventHandler<KeyboardEvent, HTMLTextAreaElement>): void {
```

Comment:
This doesn’t need to be its own function/file. I don’t think there’s any autocomplete on the title.